palantir-cassandra-jmx-api/build.gradle

```diff
@@ -38,3 +38,10 @@ task copySourceFiles(type: Copy) {
 
 tasks.withType(JavaCompile).all { dependsOn copySourceFiles }
 
+repositories {
```

Comment:
We should move BootstrapManager into an internal class so that this dependency isn't exposed on the API.